outputter/outputter.go

```diff
@@ -22,13 +22,20 @@ type Config struct {
 	Console
 	JSON
 	JSONFormat bool
+	PGSQL      bool
```

Comment:
I think I've commented on this somewhere else, I think the outputter type should be an enum rather than a series of bools because they aren't independent (e.g. I don't think you can have both JSON and Postgres, or Console & Postgres, etc)